### PR TITLE
Trader Recruit Starter Gifts

### LINKED ISSUE #30799: Trade Window Language Barriers for Recruited Friends
(WEB REPORT BY: driftingskies REMOTE: 206.221.180.138:7777)
# Revision
b575a71b92add50e00eb6a85ae1b02d0ac9ed2e4

# Description
Human's are left in the dark by the table's fancy words! There should be something to remedy this!
# Steps to Reproduce
Get a human friend, have them become a trader at the window via the PROCESS TM, have them talk to the window, profit!
# What you Expected
Them to speak a different language or the human to learn pidgin!
# What Actually Happened
The didn't!

### PR DESCRIPTION
**Pull request by Kurfursten**
closes #30799
🆑 
* rscadd: Traders recruited with a trade pact application will now receive a gift. If they speak Vox, it will be the normal trader equipment. Otherwise, they get a nanodictionary to learn Pidgin.
* rscadd: Merchants now get a copy of their licence when first greeted at the trade window.
* rscadd: Mushroom men can now purchase the mushroom-to-talon prosthetic at the trade outfitter. This will convert one hand to a robotic talon with brutal efficiency, allowing them to stamp documents.

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -620,43 +620,76 @@ var/global/list/paper_folding_results = list ( \
...
+			<body>
+			<center><img src="http://ss13.moe/wiki/images/1/17/NanoTrasen_Logo.png"> <h1>ATTN: Internal Affairs</h1></center>
+			Nanotrasen\'s commercial arm has noted the presence of a registered merchant who holds a licence for corporate commerce, a process which includes a background check and Nanotrasen loyalty implant. The associate\'s image is enclosed. Please continue to monitor trade on an ongoing basis such that Nanotrasen can maintain highest standard small business enterprise (SBE) partners.<BR>
+			<fieldset>
+	  		<legend>Picture</legend>
+			<center><img src="previewicon-[identity]1.png" width="64" height="64"><img src="previewicon-[identity]2.png" width="64" height="64"></center>
+			</fieldset><BR>
+			Name: [identity]<BR>
+			Blood Type: [merchant.dna.b_type]<BR>
+			Fingerprint: [md5(merchant.dna.uni_identity)]</body></html>"}
```
Comment (N3X15) [inline code/modules/paperwork/paper.dm:690]: If the text is going to be largely the same across these, you might consider making this an inherited function that just renders from a template.

Comment (Kurfursten) [reply code/modules/paperwork/paper.dm:690]: While the bottom part is the same, the text is subtly different and the background color is different too. If it was the whole thing I'd agree with you.

**Comment by Eneocho** [comment]
The nanodictionary's description sounds more like a weird self destructing book than a high tech one-use device. I'd take a page outta rimworld's explanations and make the nanodictionary's description a nanobot cocktail you inhale or something that directly teaches your brain one very specific thing, that would explain why you can't reuse the dictionary, the thing only had one nanobot kit. Something something you slowly inject the stuff to train, which explains the partial use.

Something like: 
"A kit of specially modified nanobots that teaches you how to speak a language. Each use consumes a sealed nanobot charge, teaching you some part of the it. Applied through the eye, once the charge reaches the brain, it quickly transforms into neural tissue as appropriate. The secondary charges are useless without the first charge, so only a single person can use the kit."

**Comment by DriftingSkies** [comment]
I'd like to see buyable nano-dictionaries via traders as well, them being random languages of course!

**Comment by Kurfursten** [comment]
Reasonable, maybe in a future crateload of them, not in this PR but to come

**Comment by Eneocho** [comment]
> 
> 
> Reasonable, maybe in a future crateload of them, not in this PR but to come

Imagine. Crewside Deathsquad speakers.

**Comment by SonixApache** [comment]
gonfligds

**Comment by Kurfursten** [comment]
Conflicts assassinated

**Pull request merged**